index.bs

```diff
@@ -555,6 +703,234 @@ Report Long Animation Frames {#loaf-processing-model}
     1. set |document|'s [=nearest same-origin root=]'s [=current frame timing info=] to null.
 </div>
 
+### Long Script Monitoring ### {#long-script-monitoring}
+
+<div algorithm="Report pending user callback">
+    To <dfn>report user callback</dfn> given a [=callback function=] |callback| and an [=environment settings object=] |settings|:
+    [=Create script entry point=] given |settings|, "`user-callback`", and the following steps given a [=script timing info=] |scriptTimingInfo|:
```

Comment:
FYI, this indent still doesn't do anything (at least, in the renderered preview); it's just part of the same paragraph as the preceding line.

Reply:
Yes, it's not supposed to do anything.